pyproject.toml

```diff
@@ -69,7 +69,7 @@ Homepage = "https://github.com/DataDog/dd-trace-py"
 "Source Code" = "https://github.com/DataDog/dd-trace-py/"
 
 [tool.setuptools_scm]
-version_scheme = "release-branch-semver"
+version_scheme = "guess-next-dev"
```

Comment:
Let's add a comment above this property explaining why this needs to be `release-branch-semver` on the main branch